Update README.md (#2193)
microsoft/BotBuilder-Samples: experimental/handoff-library/README.md

Integrations with specific agent hubs are not part of the library. 

> 

Reviewer: There is definitely a spelling mistake here. However, the grammar seems odd here. Maybe it should say something like "The library will be merged in a future release " or "The library is going to be merged..."

> The library is into be merged in a future release of the Bot Framework SDK.

Reviewer: ```suggestion
The library will be merged in a future release of the Bot Framework SDK.
```
